community-css-themes.json

```diff
@@ -358,5 +358,14 @@
         "screenshot": "screenshots/screenshot-themes-panel.png",
         "compatibleModes": ["light"],
         "branch": "main"
+    },
+    {
+        "name": "ITS Theme",
+        "author": "SlRvb",
+        "github": "SlRvb",
+        "repo": "Obsidian--ITS-Theme",

```

Comment:
Please follow the entry below for the correct `repo` format (you don't need the `github` field in this file). Thanks!